components/automate-ui/src/app/pages/service-groups/service-groups.component.ts

```diff
@@ -58,6 +58,7 @@ export class ServiceGroupsComponent implements OnInit, OnDestroy {
   private currentPage$: Observable<number>;
   private currentFieldDirection: SortDirection;
   private currentSortField: string;
+  private defaultSortField = 'percent_ok';
```

Comment:
💭 We could throw in a `readonly` here, I suppose? ([See this post for details](https://mariusschulz.com/blog/typescript-2-0-read-only-properties))

Reply:
Me gusta. 💯